### ISSUE TITLE
Feature request: interpret revscripts for creatures and spells/runes

### ISSUE THREAD

**Issue by lucasgrizante** [slawkens/myaac#189]
Current code does not support revscript system.
As otservbr and canary distros are only supporting revscripts for creatures / spells / runes, without any xml, myaac is not able to load these things.

**Comment by cssiqueira** [comment]
I'm also using canary and I'm waiting for this change in myaac, this section is really missing, in addition to the error on the items page when looking for any. :-(

**Comment by slawkens** [comment]
So, here I'm sharing some progress.

I successfully loaded a monster made in Lua by PHP script.

<img width="218" alt="image" src="https://user-images.githubusercontent.com/16687302/173831327-3eac940a-cd60-479a-9d59-310a1f93cc43.png">

Now just add loading of whole folder + saving into database.

**Comment by gesior** [comment]
I would do it other way.
All revscript engines have access to all monsters, spells and items from Lua.
I should be easier to write talkaction or globalevent 'startup' that dumps data into some table in database, than read .lua files from PHP.
Lua files format may change over time. Also multiple monsters/spells may be placed in single file, which will be even harder to read in PHP.

**Comment by slawkens** [comment]
> I would do it other way.
> All revscript engines have access to all monsters, spells and items from Lua.
> I should be easier to write talkaction or globalevent 'startup' that dumps data into some table in database, than read .lua files from PHP.
> Lua files format may change over time. Also multiple monsters/spells may be placed in single file, which will be even harder to read in PHP.

Yeah that was the idea that @gpedro also had. But do we have really access to all that things from lua? If so, writing such a script should be straightforward.

**Comment by marcelofujii** [comment]
Yo masters ! Still having this issue. Just install MyAcc + Xampp + otserv and got same problem.

**Comment by slawkens** [comment]
Hello,

So I've made the revscript loaders.

They're available in Plugins repo:
Spells - https://plugins.my-aac.org/plugins/Lua+Spells/1.0
Monsters - https://plugins.my-aac.org/plugins/Lua+Monsters/1.0
